doc/nrf/libraries/bluetooth_services/mesh/light_xyl_srv.rst

```diff
@@ -17,14 +17,108 @@ Three states can be used to configure the lighting output of an element:
+   };
+
+
+.. _bt_mesh_light_xyl_hsl_srv:
+
+Usage with the Light HSL Server
+===============================
+
+Just like the Light xyL Server, the :ref:`bt_mesh_light_hsl_srv_readme` provides the ability to control colored lights.
+In some cases, it may be desirable to use the Light HSL and Light xyL Servers to provide two different interfaces for controlling the same light.
+To acheive this, the Light xyL and Light HSL Server models should be instantiated on the same element, and share the same Lightness Server.
+
+With the Light HSL Server's corresponding :ref:`bt_mesh_light_hue_srv_readme` and :ref:`bt_mesh_light_sat_srv_readme` models, the composition data should look like this:
+
+.. table::
+   :align: center
+
+   =================  =================  =======================
+   Element N          Element N+1        Element N+2
+   =================  =================  =======================
+   Lightness Server   Light Hue Server   Light Saturation Server
+   Light HSL Server
+   Light xyL Server
+   =================  =================  =======================
+
+
+In the application code, this would look like this:
+
+.. code-block:: c
+
+   static struct bt_mesh_lightness_srv lightess_srv =
+      BT_MESH_LIGHTNESS_SRV_INIT(&lightness_cb);
+
+   static struct bt_mesh_light_hsl_srv hsl_srv =
+   	BT_MESH_LIGHT_HSL_SRV_INIT(&lightness_srv, &hue_cb, &sat_cb);
+
+   static struct bt_mesh_light_xyl_srv xyl_srv =
+ 	BT_MESH_LIGHT_XYL_SRV_INIT(&lightness_srv, &xyl_handlers);
+
+   static struct bt_mesh_elem elements[] = {
+   	BT_MESH_ELEM(
+   		1, BT_MESH_MODEL_LIST(
+            BT_MESH_MODEL_LIGHTNESS_SRV(&lightness_srv),
+            BT_MESH_MODEL_LIGHT_HSL_SRV(&hsl_srv),
+            BT_MESH_MODEL_LIGHT_XYL_SRV(&xyl_srv)),
+   		BT_MESH_MODEL_NONE),
+   	BT_MESH_ELEM(
+   		2, BT_MESH_MODEL_LIST(BT_MESH_MODEL_LIGHT_HUE_SRV(&hsl_srv.hue)),
+   		BT_MESH_MODEL_NONE),
+   	BT_MESH_ELEM(
+   		3, BT_MESH_MODEL_LIST(BT_MESH_MODEL_LIGHT_SAT_SRV(&hsl_srv.sat)),
+   		BT_MESH_MODEL_NONE),
+   };
+
+While there is just one shared instance of the Lightness Server controlling the Lightness in this configuration, the light's hue and saturation level may be controlled independently by the Light xyL and Light HSL Servers.
+The binding between the Light xyL Server's x and y states and the Light HSL Server's Hue and Saturation states is application specific.
+
+Even though there are no qualification tests verifying the binding between the color spectrum states of the Light xyL and Light HSL Servers, application developers are strongly encouraged to implement the general characteristics of bound states:
+
+* Any changes to one state should be immediately reflected in the other.
+* If publication is enabled for any of the models, a change to the bound state of one of the models should be published on both.
+* Any range limitations on one of the bound states should be respected when setting the value of the other bound state.
```

Comment:
I think there is no clear mapping of one of the states on another. If you have a limitation for `saturation` it will add range limitation both `x` and `y` due to the different coordinate system

Reply:
The opposite is the truth as well :)

Reply:
That is true. I would expect the application to be able to perform this mapping though, and implement the restriction themselves. 

In practice, I think this would go through a conversion function to fit the API their driver implements, which would apply the range limitation before pushing this to the HW. Then, the mesh status response would be a readout from the driver's current state.